src/components/screens/votingQueue/summary/summary.js

```diff
@@ -0,0 +1,148 @@
+import Piwik from '../../../../utils/piwik';
+import { PrimaryButton, SecondaryButton } from '../../../toolbox/buttons';
+import Icon from '../../../toolbox/icon';
+import ToggleIcon from '../toggleIcon';
+import VoteStats from '../voteStats';
+
+
+import styles from './styles.css';
+
+const token = tokenMap.LSK.key;
+
+const ItemList = ({ items, heading }) => (
+  <div className={styles.contentItem}>
+    <span className={styles.contentHeading}>{heading}</span>
+    <div className={styles.voteItems}>
+      {Object.keys(items).map((address, i) => (
+        <span key={i} className={styles.voteItem}>
+          <span className={styles.addressText}>{address}</span>
+          <span>
+            {Object.values(items[address]).length === 2
+              ? (
+                <>
+                  <LiskAmount val={items[address].confirmed} token={token} />
+                  <span className={styles.arrowIcon}><Icon name="arrowRightTailed" /></span>
+                  <LiskAmount val={items[address].unconfirmed} token={token} />
+                </>
+              )
+              : <LiskAmount val={Object.values(items[address])[0]} token={token} />
+            }
+          </span>
+        </span>
+      ))}
+    </div>
+  </div>
+);
+
+const InfoColumn = ({ title, children }) => (
+  <div className={styles.infoColumn}>
+    <span className={styles.infoTitle}>{title}</span>
+    <span className={styles.infoValue}>
+      {children}
+    </span>
+  </div>
+);
+
+const getResultProps = ({ added, removed, edited }) => {
+  let unlockable = Object.values(removed).reduce((sum, { confirmed }) => {
+    sum += confirmed;
+    return sum;
+  }, 0);
+
+  let locked = Object.values(added).reduce((sum, { unconfirmed }) => {
+    sum += unconfirmed;
+    return sum;
+  }, 0);
+
+  const editedWeight = Object.values(edited).reduce((sum, { confirmed, unconfirmed }) => {
+    sum += confirmed - unconfirmed;
+    return sum;
+  }, 0);
+
+  if (editedWeight > 0) {
+    locked += editedWeight;
+  } else {
+    unlockable += editedWeight;
+  }
+
+  return { locked, unlockable };
+};
+
+const Summary = ({
+  t, removed, edited, added, fee, account, prevStep, nextStep, transactions, ...props
+}) => {
+  const addedLength = Object.keys(added).length;
+  const editedLength = Object.keys(edited).length;
+  const removedLength = Object.keys(removed).length;
+
+  const {
+    locked, unlockable,
+  } = getResultProps({ added, removed, edited });
+
+  useEffect(() => {
+    if (!transactions.transactionsCreatedFailed.length
+      && transactions.transactionsCreated.length) {
+      nextStep({
+        locked, unlockable, error: false,
+      });
+    } else if (transactions.transactionsCreatedFailed.length) {
+      nextStep({
+        error: true,
+      });
+    }
+  }, [transactions]);
+
+  const submitTransaction = () => {
+    const { normalizedVotes, votesSubmitted } = props;
+    Piwik.trackingEvent('Send_SubmitTransaction', 'button', 'Next step');
```

Comment:
@reyraa This tracking code should be something else like `Send_VoteTransaction`?

Reply:
No, Let me change it. I forgot to. Thanks for catching it.